torch_xla/csrc/ops/ops.h

```diff
@@ -212,8 +212,14 @@ NodePtr BaddBmm(const Value& lhs, const Value& rhs, const Value& bias,
 
 NodePtr Lerp(const Value& start, const Value& end, const Value& weight);
 
+NodePtr LogicalNot(const Value& input);
```

Comment:
Nit: can we place these functions in their alphabetically ordered spot in this file? (i.e. after `XLATensor::logdet()`)?